tui/terminfo.lua

```diff
@@ -1158,6 +1158,14 @@ local function find_terminfo(TERM, dirs)
 	for _, dir in ipairs(dirs) do
 		local path = string.format("%s/%s/%s", dir, TERM:sub(1, 1), TERM)
 		caps, names, errno = open_terminfo(path)
+
+		-- On some system, the subdirectory is the first char's byte
+		-- See: https://invisible-island.net/ncurses/NEWS.html#t20071117
+		if caps == nil or errno == ENOENT then
+			path = string.format("%s/%s/%s", dir, string.format("%x", TERM:sub(1, 1):byte()), TERM)
```

Comment:
No need for the `:sub`, `:byte()` also takes indexes.

Reply:
Why have a double string.format? Just put the `%x` in the first one....
Also should it be `%02x`?

Reply:
Done